doc/source/whatsnew/v1.0.0.rst

```diff
@@ -1079,7 +1079,7 @@ Datetimelike
 - Bug in masking datetime-like arrays with a boolean mask of an incorrect length not raising an ``IndexError`` (:issue:`30308`)
 - Bug in :attr:`Timestamp.resolution` being a property instead of a class attribute (:issue:`29910`)
 - Bug in :func:`pandas.to_datetime` when called with ``None`` raising ``TypeError`` instead of returning ``NaT`` (:issue:`30011`)
-- Bug in :func:`pandas.to_datetime` failing for ``deques`` when using ``cache=True`` (the default) (:issue:`29403`)
+- Bug in :func:`pandas.to_datetime` failing for ``dequeues`` when using ``cache=True`` (the default) (:issue:`29403`)
```

Comment:
AFAIK this was actually a wrong suggestion of the spelling corrector. This is the plural of "dequeue" which is another word (or old spelling of deque, but python doesn't use that) -> https://github.com/pandas-dev/pandas/pull/55862